marklogic-data-hub/src/main/java/com/marklogic/hub/mapping/MappingFunctions.java

```diff
@@ -0,0 +1,22 @@
+package com.marklogic.hub.mapping;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.marklogic.client.DatabaseClient;
+import com.marklogic.client.extensions.ResourceManager;
+import com.marklogic.client.io.JacksonHandle;
+import com.marklogic.client.util.RequestParameters;
+
+public class MappingFunctions extends ResourceManager {
```

Comment:
I am seeing all the files from my PR #2957 in this PR. That PR has already been merged to feature branch

Reply:
@srinathgit , @SameeraPriyathamTadikonda - Rebased the repo to resolve the merge conflicts.